src/main/java/org/folio/circulation/rules/AppliedRuleConditionsEntity.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.circulation.rules;
+
+public class AppliedRuleConditionsEntity {
+
```

Comment:
Personally, I would remove this blank line

Reply:
removed